Add TLT minutes for February and March 2026 (#276)
ESMValGroup/Community: Tech Lead/Minutes/20260206.md

# Technical Lead Development Team call 6 February 2026

## Participants
- Attendees: @bouweandela @ehogan @schlunma @jlenh @valeriupredoi @flicj191
- Absentees: @bettina-gier @charles-turner-1 @rbeucher
- Chair: @valeriupredoi
- Minutes taker: @valeriupredoi @bouweandela (retrospect)
  - Names legend:
    - Bouwe Andela [@bouweandela](https://github.com/bouweandela)

>     - Saskia Loosveldt [@sloosvel](https://github.com/sloosvel)

Reviewer: ```suggestion
```

    - Romain Beucher [@rbeucher](https://github.com/rbeucher)
    - Felicity Chun [@flicj191](https://github.com/flicj191)
    - Charles Turner [@charles-turner-1](https://github.com/charles-turner-1)
    - Julien Lenhardt [@jlenh](https://github.com/jlenh)

## Pending Action Items
| Meeting | Assignees | Item | Status |
|-|-|-|-|

## Agenda

- release 2.14 update: @schlunma (with @jlenh and @flicj191 as release buddies):
  - ESMValCore: couple of minor PRs still to be merged (today), one big one #2918 that is still to be decided if merged for 2.14
  - ESMValCore RC ideally to be rolled out start next week (9-13 February)
  - @ehogan asking about the new config structure if is there is still some backwards functionality; @bouweandela giving an overview about that, and what it'll be included in release 2.14
  - @schlunma: reading non-CMOR standard data? @bouweandela: yes so long as it is CF-compliant, possible skip all checks and simply ingest the data into preprocessing; Bouwe also discussing the need for CMOR standrds for ESMValTool (need to have strict standards for inter-comparison)
  - @bouweandela suggesting to use the Release Slack channel!

>   - ESMValTool Milestones: few bookeeping (renaming files etc), couple cmorizers, one MO diagnostic, ESA-related cmorizers that @schlunma will ask Axel - **we should do tech review** - @jlenh, @flicj191 "volunteered" - thank you, @bouweandela looking at distributing them

Reviewer: ```suggestion
  - ESMValTool Milestones: few bookkeeping (renaming files etc), couple cmorizers, one MO diagnostic, ESA-related cmorizers that @schlunma will ask Axel - **we should do tech review** - @jlenh, @flicj191 "volunteered" - thank you, @bouweandela looking at distributing them
```

- email response: tech support for training sessions organized by Ranjini - who will be there, and if we can provide support: @bouweandela will be at least at the one after the Workshop, @valeriupredoi will be too

> - @bouweandela: showing us traffic metrics from visitors to ESMValCore/Tool documentation and GitHub pages (newly implemented via Matomo)

Reviewer: ```suggestion
- @bouweandela: showing us traffic metrics from visitors to ESMValCore/Tool documentation and tutorial and website (newly implemented via Matomo)
```
